packages/react-scripts/config/webpack.config.js

```diff
@@ -737,6 +757,11 @@ module.exports = function (webpackEnv) {
         resolvePluginsRelativeTo: __dirname,
         baseConfig: {
           extends: [require.resolve('eslint-config-react-app/base')],
+          rules: {
+            ...(!hasJsxRuntime && {
```

Comment:
This would prevent someone from turning off the rule I believe. Could that be an issue in the case that someone has opted out of the newer runtime?

Reply:
`hasJsxRuntime` takes the disable flag into account so I think this should be okay.

Reply:
Do you mean if they opt out *and* want to turn this rule off they can't?

Reply:
I just tested that scenario and it is possible to turn the rule off.